packages/block-editor/src/hooks/use-color-props.js

```diff
@@ -0,0 +1,118 @@
+/**
```

Comment:
Weren't this merged as part of https://github.com/WordPress/gutenberg/pull/30869? I guess this needs a rebase.

Reply:
✅ Rebased.